release-notes/9.0/preview/preview7/sdk.md

```diff
@@ -47,24 +47,24 @@ You can see what mode your SDK installation is in by running `dotnet workload --
 
 ```terminal
 > dotnet workload --info
-Workload version: 9.0.100-manifests.400dd185
+ Workload version: 9.0.100-preview.7.24414.1
 Configured to use loose manifests when installing new manifests.
  [aspire]
-   Installation Source: VS 17.10.35027.167, VS 17.11.35111.106
-   Manifest Version:    8.0.2/8.0.100
-   Manifest Path:       C:\Program Files\dotnet\sdk-manifests\8.0.100\microsoft.net.sdk.aspire\8.0.2\WorkloadManifest.json
-   Install Type:              Msi
+   Installation Source: SDK 8.0.400
```

Comment:
is this a valid result for .NET 9 preview 7?

Reply:
If the aspire workload was last installed when using 8.0.400-band SDKs, then yes - the next time a workload update happened (via `dotnet workload install` or `dotnet workload update`) I'd expect this to change to align with the in-use SDK's feature band.